### ISSUE TITLE
Autocompletion doesn't work for filename.blade.php File Format

### ISSUE THREAD

**Issue by vindos** [redgluten/laravel_forms_boostrap_snippets#6]
If the file name is 'filename.blade.php', then the autocompletion is not working. I have checked the 'snippet files'. In the 'scope' section if we add 'source.php', then the auto-completion started working.

**<scope>text.blade,text.html.laravel-blade,_source.php_</scope>**

**Comment by redgluten** [comment]
It does for me but it seems that scopes are changing a lot with ST lately, which version are you using?

**Comment by zcythe** [comment]
Sublime3 build 3103

**Comment by jayleco** [comment]
Sublime3 build 3114
this plugin is not working in the  XX.blade.php file
Thanks Auth!

**Comment by jayleco** [comment]
Too need this plugin

**Comment by redgluten** [comment]
I added `,source.php` to the snippet’s scope. Tell me if it works OK for you because it never stopped working for me.

**Comment by vindos** [comment]
My Version is '**Sublime 3 Build 3083**'.
As told by **@redgluten** rightly, the scope is getting changed a lot.
I have modified the snippet files by adding 'source.php' then auto-completion worked within the tags <?php and ?>

But in the 'filename.blade.php' files, it never worked.
On checking, got to know that by pressing the  **'Ctrl+Shift+Alt+P'** will show the 'Scope' at the bottom of the Sublime. Based on that, I realized that it is 'text.html.basic'

Hence I have even added the scope as '**`text.html.basic`**' and all the plugins started working. 
Hope this helps others having the same issue!!
